prusti-server/src/server.rs

```diff
@@ -41,61 +41,53 @@ pub fn spawn_server_thread() -> SocketAddr {
     receiver.recv().unwrap()
 }
 
+// initialize on first access
```

Comment:
The comment is implied by the `Lazy` type. Can you instead summarize what this global does?